src/data/mv_polynomial.lean

```diff
@@ -20,12 +23,10 @@ def mv_polynomial (σ : Type*) (α : Type*) [comm_semiring α] := (σ →₀ ℕ
 namespace mv_polynomial
 variables {σ : Type*} {a a' a₁ a₂ : α} {e : ℕ} {n m : σ} {s : σ →₀ ℕ}
 
-variables [decidable_eq σ] [decidable_eq α]
-
 section comm_semiring
 variables [comm_semiring α] {p q : mv_polynomial σ α}
 
-instance : decidable_eq (mv_polynomial σ α) := finsupp.decidable_eq
+def decidable_eq_mv_polynomial [decidable_eq σ] [decidable_eq α] : decidable_eq (mv_polynomial σ α) := finsupp.decidable_eq
```

Comment:
idem.

Reply:
idem.

Reply:
Okay, this one doesn't cause problems, so I've restored it.